package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leetcode-sync",
-  "version": "1.0.0",
+  "version": "1.4.0",
```

Comment:
Maybe also revert this (although I think this one doesn't matter as much, shouldn't break anything if it does change AFAIK), I can update it after I make the new release